src/main/java/com/airlines/model/CrewMember.java

```diff
@@ -100,11 +112,11 @@ public boolean equals(Object o) {
             return false;
         }
         CrewMember that = (CrewMember) o;
-        return getId() == that.getId() && Objects.equals(getFirstName(), that.getFirstName()) && Objects.equals(getLastName(), that.getLastName()) && getPosition() == that.getPosition() && Objects.equals(getBirthday(), that.getBirthday());
+        return getId() == that.getId() && getCrewId() == that.getCrewId() && Objects.equals(getFirstName(), that.getFirstName()) && Objects.equals(getLastName(), that.getLastName()) && getPosition() == that.getPosition() && Objects.equals(getBirthday(), that.getBirthday());
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(getId(), getFirstName(), getLastName(), getPosition(), getBirthday());
+        return Objects.hash(getId(), getCrewId(), getFirstName(), getLastName(), getPosition(), getBirthday());
```

Comment:
anyway, I suggest using field order in hashcode/equals, so crewId should be last